yarn.lock

```diff
@@ -1679,6 +1679,37 @@
     lodash "^4.17.19"
     to-fast-properties "^2.0.0"
 
+"@backstage/catalog-client@^0.3.11":
+  version "0.3.11"
+  resolved "https://registry.npmjs.org/@backstage/catalog-client/-/catalog-client-0.3.11.tgz#3454a26b763afe5a25aac92e4503f19e7ad6f607"
+  integrity sha512-AgMXP9BwHMBSIuBNdcluA0hWmHqRrDLvovwY/hxX7FbD1t4s5RzekHb5VyI6UaxcxBL+9agu+b23Vax5u9jIKQ==
+  dependencies:
+    "@backstage/catalog-model" "^0.7.8"
+    "@backstage/config" "^0.1.5"
+    "@backstage/errors" "^0.1.1"
+    cross-fetch "^3.0.6"
+
+"@backstage/catalog-model@^0.7.8":
+  version "0.7.8"
+  resolved "https://registry.npmjs.org/@backstage/catalog-model/-/catalog-model-0.7.8.tgz#3226d2416caa12bad4ca8ebca31218e469572f89"
+  integrity sha512-YPoSdhzc1T2SisKI4G2HyrLaRM4xqEEN0iayPydpjq8ki5CBVmQTHi6Qv5rYa9AYjA8io7EwgSGl7c5QWc4HWw==
+  dependencies:
+    "@backstage/config" "^0.1.5"
+    "@types/json-schema" "^7.0.5"
+    "@types/yup" "^0.29.8"
+    ajv "^7.0.3"
+    json-schema "^0.3.0"
+    lodash "^4.17.15"
+    uuid "^8.0.0"
+    yup "^0.29.3"
+
+"@backstage/config@^0.1.5":
+  version "0.1.5"
+  resolved "https://registry.npmjs.org/@backstage/config/-/config-0.1.5.tgz#a883c1b5bdab7ab2ff041cd278f3e8c885203cb3"
+  integrity sha512-/pudFgexXqFLe1NdLDVeosBfJwS6+VsuNAduXW+g4901BxwptojqJxJcpi42nC2F9z74jDtYJlqlYIsJAEB8nA==
+  dependencies:
+    lodash "^4.17.15"
```

Comment:
these three entries above are a sign that the version depended on does not match the actual in-repo version of those packages. This made yarn pull in duplicates from NPM, which will cause problems. So you'll want to update your version ranges in your package.json to `^` plus the exact version